pkg/pki/x509/x509.go

```diff
@@ -188,6 +188,13 @@ func (k PublicKey) EmailAddresses() []string {
 				names = append(names, strings.ToLower(name))
 			}
 		}
+		for _, name := range cert.URIs {
+			validate := validator.New()
+			errs := validate.Var(name.String(), "required,uri")
+			if errs == nil {
+				names = append(names, strings.ToLower(name.String()))
+			}
+		}
```

Comment:
nit:
```suggestion
			if errs := validate.Var(name.String(), "required,uri"); errs == nil {
				names = append(names, strings.ToLower(name.String()))
			}
		}
```

Reply:
Are these validation errors surfaced anywhere before/after this? It seems a waste to drop perfectly good validation errors from end users (...says the end user most likely to send invalid values 🙃 )

Reply:
This ones a little tougher to surface -- I expect that this one is relatively difficult to hit, because Fulcio is the one populating certificates and it's not user-controlled. 

On that note maybe it's worth filing an issue that we surface what keys WERE successfully indexed when you upload something?

Reply:
FWIW, it's also worth noting that it might not be Fulcio's certs uploaded to the log, as users can upload certs from their own PKI.